relayer/compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+  bitcoind:
+    image: ruimarinho/bitcoin-core:24.0.1
```

Comment:
There are more maintainable images
![image](https://github.com/user-attachments/assets/7678de78-e65e-4db3-971a-c898822b02f2)
eg this one https://hub.docker.com/r/lncm/bitcoind

Reply:
or https://github.com/willcl-ark/bitcoin-core-docker

Reply:
[lncm/bitcoind](https://hub.docker.com/r/lncm/bitcoind) looks good. It seems to be working without any other changes, though I of course didn't run the full sync like with the previous one.

498f3d8df4d5c5f1b86e3f888b6cf669e08830e0